packages/protoc-gen-connect-es/README.md

```diff
@@ -147,6 +147,16 @@ empty files, to allow for smooth interoperation with Bazel and similar
 tooling that requires all output files to be declared ahead of time.
 Unless you use Bazel, it is very unlikely that you need this option.
 
+### `ts_nocheck=false`
+
+By default, [protoc-gen-connect-es](https://www.npmjs.com/package/@connectrpc/protoc-gen-connect-es)
+(and all other plugins based on [@bufbuild/protoplugin](https://www.npmjs.com/package/@bufbuild/protoplugin))
+generate an annotation at the top of each file: `// @ts-nocheck`.
+
+We generate the annotation to support a wide of compiler configurations and
```

Comment:
nit: `a wide {range, array, etc.?} of compiler configurations...`

Reply:
Thanks for actually reviewing 👍